### ISSUE TITLE
[bug] vite + pnpm not working

### ISSUE THREAD

**Issue by astahmer** [tamagui/tamagui#355]
hey, tamagui looks amazing, I wanted to try it using `vite` so I followed [the guide](https://tamagui.dev/docs/guides/vite) and added both plugins (classic+compiler) to vite.config + created the tamagui.config.ts and added the provider, sadly it crashes instantly when visiting the page

---

**Describe the bug**

getting this error when visiting [localhost](http://localhost:5173/)

```
chunk-ZD4SVSDO.js?v=f2f5fd50:4858 Uncaught TypeError: Cannot destructure property 'themes' of 'getConfig(...)' as it is undefined.
    at TamaguiProvider (chunk-ZD4SVSDO.js?v=f2f5fd50:4858:11)
    at renderWithHooks (chunk-GZW3ZYR6.js?v=f2f5fd50:11761:26)
    at mountIndeterminateComponent (chunk-GZW3ZYR6.js?v=f2f5fd50:14490:21)
    at beginWork (chunk-GZW3ZYR6.js?v=f2f5fd50:15445:22)
    at HTMLUnknownElement.callCallback2 (chunk-GZW3ZYR6.js?v=f2f5fd50:3573:22)
    at Object.invokeGuardedCallbackDev (chunk-GZW3ZYR6.js?v=f2f5fd50:3598:24)
    at invokeGuardedCallback (chunk-GZW3ZYR6.js?v=f2f5fd50:3632:39)
    at beginWork$1 (chunk-GZW3ZYR6.js?v=f2f5fd50:19258:15)
    at performUnitOfWork (chunk-GZW3ZYR6.js?v=f2f5fd50:18691:20)
    at workLoopSync (chunk-GZW3ZYR6.js?v=f2f5fd50:18630:13)
```

**To Reproduce**

- `pnpm i`
- `pnpm dev`
- 
https://github.com/astahmer/vite-react-tamagui-ts

**System Info (please complete the following information):** 

System:
    OS: macOS 12.5.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 437.83 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.0 - ~/Library/Caches/fnm_multishells/29730_1668207089892/bin/node
    Yarn: 1.22.19 - ~/Library/pnpm/yarn
    npm: 8.3.1 - ~/Library/Caches/fnm_multishells/29730_1668207089892/bin/npm
  Browsers:
    Chrome: 107.0.5304.110
    Firefox: 104.0
    Safari: 15.6.1
  npmPackages:
    @astahmer/eslint-config-react: ^0.1.5 => 0.1.5 
    @babel/types: ^7.20.2 => 7.20.2 
    @iconify/json: ^2.1.136 => 2.1.136 
    @tamagui/config-base: 1.0.1-beta.214 => 1.0.1-beta.214 
    @tamagui/core: 1.0.1-beta.214 => 1.0.1-beta.214 
    @tamagui/shorthands: 1.0.1-beta.214 => 1.0.1-beta.214 
    @tamagui/theme-base: 1.0.1-beta.214 => 1.0.1-beta.214 
    @tamagui/vite-plugin: 1.0.1-beta.214 => 1.0.1-beta.214 
    @types/react: ^18.0.25 => 18.0.25 
    @types/react-dom: ^18.0.8 => 18.0.8 
    @typescript-eslint/eslint-plugin: ^5.42.1 => 5.42.1 
    @unocss/preset-icons: ^0.46.4 => 0.46.4 
    @vitejs/plugin-react: ^2.2.0 => 2.2.0 
    eslint: ^8.27.0 => 8.27.0 
    eslint-formatter-pretty: ^4.1.0 => 4.1.0 
    pastable: ^2.0.13 => 2.0.13 
    prettier: ^2.7.1 => 2.7.1 
    react: ^18.2.0 => 18.2.0 
    react-dom: ^18.2.0 => 18.2.0 
    react-error-boundary: ^3.1.4 => 3.1.4 
    react-native-web: ^0.18.10 => 0.18.10 
    tamagui: 1.0.1-beta.214 => 1.0.1-beta.214 
    ts-pattern: ^4.0.5 => 4.0.5 
    ts-toolbelt: ^9.6.0 => 9.6.0 
    type-fest: ^3.2.0 => 3.2.0 
    typescript: ^4.8.4 => 4.8.4 
    unocss: ^0.46.4 => 0.46.4 
    vite: ^3.2.3 => 3.2.3 
    vite-plugin-checker: ^0.5.1 => 0.5.1 
    vite-plugin-compress: ^2.1.1 => 2.1.1 

**Tamagui info**

Paste all the following below:

- [x] pnpm-lock.yaml is in the repo linked above
- [x] no specific file
- [x] no logs output with `DEBUG=tamagui`

**Comment by natew** [comment]
Fixed in a701eb3f8c83f96d9ddce419388fdd710b8fa6c4 (next release will have it)

**Comment by natew** [comment]
In beta 215, reopen if still an issue
